rust/crd/src/lib.rs

```diff
@@ -1298,38 +1294,6 @@ spec:
         );
     }
 
-    #[test]
```

Comment:
Removed because of serde_yaml bump serializes this using yaml tags and I don't see a big enough benefit in having this test to rewrite it. Feel free to veto ;)

Reply:
true. this must have been part of an experiment.